custom/enikshay/ucr/data_sources/person_for_referral_report_v3.json

```diff
@@ -17,19 +17,14 @@
         "table_id": "person-for-referral-v3",
         "display_name": "person (for referral report) V3",
         "configured_filter": {
-            "type": "and",
-            "filters": [
-                {
-                    "operator": "eq",
-                    "expression": {
-                        "datatype": null,
-                        "type": "property_name",
-                        "property_name": "type"
-                    },
-                    "type": "boolean_expression",
-                    "property_value": "person"
-                }
-            ]
+            "operator": "eq",
```

Comment:
I think this will cause a rebuild for this table, right? I think you should leave this as is then, since it works, and that's the pattern we've been using in the other datasources anyway.

Reply:
oh, didnt realize that would be in here. i merged my branch from https://github.com/dimagi/commcare-hq/pull/18792 to prevent a merge conflict on the other data source. I thought it was already merged, i see it was just approved. but this pattern will make rebuilds work better. @emord can explain better, but basically when there is a simple filter (ie no and) we can only add the right case types to the queue for the rebuild as opposed to going through all cases. this also shouldnt trigger a rebuild either way. I think we only do that when adding a new column. and all the new ones are in place anyway.

Reply:
as mentioned earlier there's no rebuild for filter changes